runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/BindingConfig.java

```diff
@@ -40,8 +42,8 @@ public class BindingConfig
     public final String entry;
     public final String vault;
     public final OptionsConfig options;
+    public final ObjectHashSet<CatalogedConfig> catalogs;
```

Comment:
Try to avoid using implementation details in fields, let's use `Set<CatalogedConfig>` or `Collection<CatalogedConfig>` instead.